app/src/main/res/values/strings.xml

```diff
@@ -268,7 +268,7 @@
     <!-- Home onboarding dialog welcome screen title text. -->
     <string name="onboarding_home_welcome_title">Welcome to an independent internet</string>
     <!-- Home onboarding dialog welcome screen description text. -->
-    <string name="onboarding_home_welcome_description">Pick up where you left off with tabs from other devices now on your homepage.</string>
+    <string name="onboarding_home_welcome_description">More colors. Better privacy. Same commitment to people over profilts.</string>
```

Comment:
```suggestion
    <string name="onboarding_home_welcome_description">More colors. Better privacy. Same commitment to people over profits.</string>
```

Reply:
I think even the design should have the word `profit`

Reply:
Good catch, it looks like the design spelt it wrong too. 
<img width="672" alt="Screen Shot 2022-08-25 at 10 45 47 PM" src="https://user-images.githubusercontent.com/1190888/186805642-770b2077-57e8-44d0-8d49-419f42c935df.png">